src/Components/Detail/Detail.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import './Detail.scss';
+import '../../Styles/common.scss';
+import '../../Styles/reset.scss';
+
+class Detail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      product: {},
+    };
+  }
+  getInfo = () => {
+    fetch('/data/mockdata.json', {
+      method: 'GET',
+    })
+      .then(products => products.json())
+      .then(products => {
+        this.setState({
+          product: products.filter(product => product.id === 2),
+        });
+      });
+  };
+  componentDidMount() {
+    this.getInfo();
+  }
+  render() {
```

Comment:
```suggestion
  constructor(props) {
    super(props);
    this.state = {
      product: {},
    };
  }
  
  getInfo = () => {
    fetch('/data/mockdata.json', {
      method: 'GET',
    })
      .then(products => products.json())
      .then(products => {
        this.setState({
          product: products.filter(product => product.id === 2),
        });
      });
  };
  
  componentDidMount() {
    this.getInfo();
  }
  
  render() {
```
줄바꿈을 통해 가독성을 높여볼까요? :)

Reply:
넵 알겠습니다!